src/ext/DifxApp/wixext/WixToolset.DifxApp.wixext.targets

```diff
@@ -7,5 +7,9 @@
        Condition=" Exists('$(MSBuildThisFileDirectory)..\$(WixToolsetExtensionPackageFolder)\$(MSBuildThisFileName).dll') " />
     <UnsupportedWixExtension Include="$(MSBuildThisFileName)"
       Condition=" !Exists('$(MSBuildThisFileDirectory)..\$(WixToolsetExtensionPackageFolder)\$(MSBuildThisFileName).dll') " />
+    <WixLibrary Include="$(MSBuildThisFileDirectory)..\$(WixToolsetExtensionPackageFolder)\difxapp_x86.wixlib"
```

Comment:
What if we just put `$(Platform)` in the filename here? That would get us down to one `WixLibrary`. It would also fail with the appropriate error message when someone uses a `$(Platform)` that difx doesn't support.